src/main/java/com/example/bcsd/controller/ArticleController.java

```diff
@@ -0,0 +1,55 @@
+package com.example.bcsd.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.bcsd.dto.RequestDto.ArticleCreateRequestDto;
+import com.example.bcsd.dto.RequestDto.ArticleUpdateRequestDto;
+import com.example.bcsd.dto.ResponseDto.ArticleResponseDto;
+import com.example.bcsd.service.ArticleService;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+
+
+
+@RestController
+@RequestMapping("/articles")
+public class ArticleController {
+    
+    @Autowired
+    private ArticleService articleService;
+
+    @GetMapping("/{id}")
+    public ResponseEntity<ArticleResponseDto> getArticle(@PathVariable Long id){
+        ArticleResponseDto articleDto = articleService.getArticle(id);
+        return ResponseEntity.ok().body(articleDto);
+    }
+
+    @PostMapping()
+    public ResponseEntity<ArticleResponseDto> createArticle(@RequestBody ArticleCreateRequestDto requestDto){
+        ArticleResponseDto articleDto = articleService.createArticle(requestDto.getTitle(), requestDto.getContent());
```

Comment:
```suggestion
        ArticleResponseDto response = articleService.createArticle(requestDto.getTitle(), requestDto.getContent());
```
처럼 DTO의 이름을 response라고 적는게 더 "반환값"의 의미에 가까워서 좋을 거 같아요.